app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+import os
+import re
+
+
+def move_file(command: str) -> None:
+    try:
+        command, source_path, destination_path = command.split()
+    except ValueError:
+        print("Wrong command format")
+    else:
+        if command == "mv" and source_path != destination_path:
+            if not os.path.isfile(source_path):
+                raise ValueError(f"File '{source_path}' doesn't exist")
+            source_path = os.path.join(*re.split(r"[\\/]", source_path))
```

Comment:
os.path.split() is prefereable method to correctly split file path on all systems